src/test/java/study/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,66 @@
+package study;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+class StringAddCalculatorTest {
+    private StringAddCalculator stringAddCalculator;
+
+    @BeforeEach
+    void setup() {
+        stringAddCalculator = new StringAddCalculator();
+    }
+
+    @ParameterizedTest
+    @DisplayName("null 또는 빈 문자열 입력시 splitAndSum 체크")
+    @NullAndEmptySource
+    public void splitAndSum_null_또는_빈문자(String text) {
+        int result = stringAddCalculator.splitAndSum(text);
+        assertThat(result).isEqualTo(0);
+
+        result = stringAddCalculator.splitAndSum(text);
+        assertThat(result).isEqualTo(0);
```

Comment:
`@NullAndEmptySource` 👍
테스트 구동 시, 메서드 파라미터인 `text`에 각각 `null`과 `""` 값이 할당되어 수행되기 때문에
다음과 같이 줄여볼 수 있을 것 같아요.
```suggestion
        int result = stringAddCalculator.splitAndSum(text);
        assertThat(result).isZero();
```